server/config.go

```diff
@@ -98,21 +143,36 @@ func (s *Service) ReplaceConfigSection(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	section := httprouter.GetParamFromContext(ctx, "section")
 	var res interface{}
-	switch section {
-	case "auth":
-		var authConfig chronograf.AuthConfig
-		if err := json.NewDecoder(r.Body).Decode(&authConfig); err != nil {
-			invalidJSON(w, s.Logger)
-			return
-		}
-		config.Auth = authConfig
-		res = newAuthConfigResponse(*config)
-	default:
-		Error(w, http.StatusBadRequest, fmt.Sprintf("received unknown section %q", section), s.Logger)
+	res = newAuthConfigResponse(*config)
```

Comment:
ditto here. Don't think we need to use an empty interface.

Reply:
👍